test/k8s-integration/main.go

```diff
@@ -344,9 +434,24 @@ func pushImage(pkgDir, stagingImage, stagingVersion string) error {
 
 func deleteImage(stagingImage, stagingVersion string) error {
 	cmd := exec.Command("gcloud", "container", "images", "delete", fmt.Sprintf("%s:%s", stagingImage, stagingVersion), "--quiet")
-	err := runLongRunningCommand("Deleting GCR Container", cmd)
+	err := runCommand("Deleting GCR Container", cmd)
 	if err != nil {
 		return fmt.Errorf("failed to delete container image %s:%s: %s", stagingImage, stagingVersion, err)
 	}
 	return nil
 }
+
+func fillinOverlayTemplate(overlayDir, stagingImage string) error {
+	// Substitute the PROW_GCEPD_IMAGE env with stagingImage for every file in the directory
```

Comment:
hmmm I was hoping `kustomize` would help solve this problem somehow. Is there no way to use environment variables or flags to kustomize to use built in functionality for this replace type stuff.

Reply:
Kustomize doesn't support changing the image registry dynamically yet: https://github.com/kubernetes-sigs/kustomize/issues/402 

I have a comment earlier that once this is fixed, then we can remove this substitution part.